frontend/common/kubeflow-common-lib/projects/kubeflow/src/lib/status-icon/status-icon.component.scss

```diff
@@ -1,15 +1,24 @@
-.small-padding-right {
-  padding-right: 8px;
+.align-middle {
+  vertical-align: middle;
+}
+
+// TODO: snack-bar uses the same colors. you can use a global styles or better use css variables
+.check_circle {
+  color: green;
+}
+
+.unavailable {
```

Comment:
This is never used? I see that timelapse is used, which is the same color, but don't see a ref to this one.